solr/contrib/prometheus-exporter/conf/grafana-solr-dashboard.json

```diff
@@ -226,6 +226,131 @@
         "alignLevel": null
       }
     },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "Prometheus",
+      "description": "",
+      "fieldConfig": {
+        "defaults": {
+          "unit": "short"
+        },
+        "overrides": []
+      },
+      "fill": 0,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 8,
+        "w": 6,
+        "x": 6,
+        "y": 1
+      },
+      "hiddenSeries": false,
+      "id": 221,
+      "legend": {
+        "alignAsTable": false,
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 2,
+      "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
+      "percentage": false,
+      "pluginVersion": "7.5.4",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [
+        {
+          "$$hashKey": "object:395",
+          "alias": "Ensemble size",
+          "lines": true
+        },
+        {
+          "$$hashKey": "object:396",
+          "alias": "Healthy nodes",
+          "bars": true,
+          "color": "#5794F2",
+          "lines": false
+        }
+      ],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "exemplar": true,
```

Comment:
@thelabdude I noticed in an environment I was running this, that the Zookeeper graph was not working. Playing around in Grafana UI, I noticed that the "Examplar" buttons were checked for this panel only, and unchecking them made the graph appear again.

As I understand, Exemplars is some sort of distributed tracing support in Prometheus. I think we should remove this setting in solr git. Wdyt?